mdtraj/formats/amberrst.py

```diff
@@ -462,7 +463,7 @@ class AmberNetCDFRestartFile(object):
     def __init__(self, filename, mode='r', force_overwrite=False):
         self._closed = True
         self._mode = mode
-        if StrictVersion(import_('scipy.version').short_version) < StrictVersion('0.12.0'):
+        if Version(import_('scipy.version').short_version) < Version('0.12.0'):
```

Comment:
(don't need to fix here)

This check is hilariously old, can safely be dropped now: 
<img width="451" alt="image" src="https://github.com/mdtraj/mdtraj/assets/7935382/36353a6f-5656-48c8-a104-5892ad9821c1">